server/ansible/roles/pbench-server-vars/tasks/main.yml

```diff
@@ -0,0 +1,162 @@
+---
+# Get variables from the config file on the server and register them
+# as facts, so they can be used in all subsequent roles. It is assumed
+# that the config file has already been installed on the server.
+
+# pbench top level directory
+- name: fetch the top-level pbench directory from remote
+  shell: getconf.py pbench-top-dir pbench-server
+  register: cmd_output
+  environment:
+    CONFIG: /opt/pbench-server/lib/config/pbench-server.cfg
```

Comment:
Is the `/opt/pbench-server` configurable?

Reply:
See above.